docs/manual/parallel_algorithms.qbk

```diff
@@ -271,6 +271,16 @@ __hpx__ provides implementations of the following parallel algorithms:
      [`<hpx/include/parallel_set_operations.hpp>`]]
 ]
 
+[table Heap operations (In Header: <hpx/include/parallel_algorithm.hpp>)
+    [[Name]     [Description]   [In Header]]
+    [[ [algoref is_heap] ]
+     [Returns `true` if the range is max heap.]
```

Comment:
Hmm.. But, I think using word "max heap' is right. With std::less as a predicate, biggest element is root of heap.